html_css/scripts/pagination.js

```diff
@@ -0,0 +1,67 @@
+export function paginator(selector, data) {
+	initPaginator(data, selector)
+}
+function initPaginator(data, selector) {
```

Comment:
In general, you could do delegate events, that is, just listen to the parent element, and look at which child element was clicked on, this will dramatically reduce the number of listeners to the event.